pytests/tests/test_json.py

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright (C) 2022 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the GNU General Public License v2 (the "License");
+# you may not use this file except in compliance with the License. The terms
+# of the License are located in the COPYING file of this distribution.
+#
+
+import pytest
+import json
+
+
+@pytest.fixture(scope='function', autouse=True)
+def setup_test(utils):
+    yield
+    teardown_test(utils)
+
+
+def teardown_test(utils):
+    pkgname = utils.config["mulversion_pkgname"]
+    utils.erase_package(pkgname)
+
+
+def test_list(utils):
+    ret = utils.run(['tdnf', '-j', 'list'])
+    infolist = json.loads("\n".join(ret['stdout']))
+
+    glibc_found = False
+    for info in infolist:
+        if info['Name'] == "glibc":
+            glibc_found = True
+            break
+    assert(glibc_found)
+
+
+def test_info(utils):
+    ret = utils.run(['tdnf', '-j', 'info'])
+    infolist = json.loads("\n".join(ret['stdout']))
+
+    glibc_found = False
+    for info in infolist:
+        if info['Name'] == "glibc":
+            glibc_found = True
```

Comment:
just return instead of break, if we go out of loop, then assert false.
Applies in all places, generic comment
```
    for info in infolist:
        if info['Name'] == "glibc":
            return
    assert(False)
```